meson.build

```diff
@@ -83,6 +83,10 @@ foreach header : ['pwd.h', 'strings.h', 'netinet/in.h', 'sys/socket.h']
   endif
 endforeach
 
+if cxx.get_id() == 'msvc'
+  conf_data.set('NOMINMAX', true)
+endif
+
 if host_machine.endian() == 'big'
   conf_data.set('WORDS_BIGENDIAN', 1)
```

Comment:
Do you know if this also controls non-dosbox code behavior too, like 3rd party headers?
Would be nice if this fell in line with C_WORDS_BIGENDIAN syntax (if it makes sense).

Reply:
AFAIK `WORDS_BIGENDIAN` macro is an autoconf invention (because there's no standard macros for defining endianness in C/C++ compilers) I don't think it affects 3rd party headers (unless those projects use autoconf and have badly written public headers).

I consider this different from `C_*` macros - it's not a user-selectable feature, but a property of target CPU. Let's keep it as it is.